File: backend/express/models/ticket.js
const mongoose = require('mongoose');

const ticketSchema = new mongoose.Schema({
  movieTitle: { type: String, required: true },
  seatNumber: { type: String, required: true },
  purchaseTime: { type: Date },
  showTime: { type: Date, required: true },
});

module.exports = mongoose.model('Ticket', ticketSchema);

File: backend/aspnet/Models/Ticket.cs
namespace aspnet.Models;
using System.ComponentModel.DataAnnotations;

public class Ticket
{
    public int Id { get; set; }
    [Required]
    public string MovieTitle { get; set; }
    [Required]
    public string SeatNumber { get; set; }
    [Required]
    public DateTime ShowTime { get; set; }
}